source/georef/street_network.cpp

```diff
@@ -573,8 +575,8 @@ Path create_path(const GeoRef& geo_ref,
         last_transport_carac = transport_carac;
         path_item.way_idx = edge.way_idx;
         path_item.transportation = transport_carac;
-        path_item.duration += edge.duration;
-        p.duration += edge.duration;
+        path_item.duration += edge.duration / speed_factor;
+        p.duration += path_item.duration;
```

Comment:
oups you're adding to much duration here, not a good thing that the UT pass :wink: